src/main/java/org/owasp/esapi/reference/DefaultValidator.java

```diff
@@ -102,16 +131,22 @@ public static Validator getInstance() {
 
     /**
      * Default constructor uses the ESAPI standard encoder for canonicalization.
+     * This uses an {@code Encoder} created based on the {@code Codec}s
+     * specified by the value of the {@code Encoder.DefaultCodecList} ESAPI
+     * property as defined in your <b>ESAPI.properties</b> file.
      */
     public DefaultValidator() {
         this.encoder = ESAPI.encoder();
     }
 
     /**
      * Construct a new DefaultValidator that will use the specified
-     * Encoder for canonicalization.
-     *
-     * @param encoder
+     * {@code Encoder} for canonicalization.
+     * @param encoder The specialy constructed ESAPI {@code Encoder} instance
```

Comment:
should specialy have two "l"s and not one?

I'm British, so it could just be a US v British English spelling difference I'm unfamiliar with

Reply:
Good catch.